CHANGELOG.md

```diff
@@ -298,6 +313,7 @@
 * Code Style: Add branches around single line blocks. [#13097](https://github.com/mixxxdj/mixxx/pull/13097)
 * Add missing member in copy ctor [#13229](https://github.com/mixxxdj/mixxx/pull/13229)
 * Refactor/preferences enums [#12798](https://github.com/mixxxdj/mixxx/pull/12798)
+* (opti/cleanup) make use of WLibrary::getCurrentTrackTableView() [#13335](https://github.com/mixxxdj/mixxx/pull/13335)
```

Comment:
I used that prefix only as tag, let's remove it.
And move that PR to the Library section?
```suggestion
* Library control: make use of WLibrary::getCurrentTrackTableView() [#13335](https://github.com/mixxxdj/mixxx/pull/13335)
```